src/frontend/typer/vedtak/vedtak.ts

```diff
@@ -31,9 +31,12 @@ export enum ÅrsakAvslag {
     RETT_TIL_UTSTYRSSTIPEND = 'RETT_TIL_UTSTYRSSTIPEND',
     HAR_IKKE_MERUTGIFTER = 'HAR_IKKE_MERUTGIFTER',
     RETT_TIL_BOSTØTTE = 'RETT_TIL_BOSTØTTE',
+    REISEAVSTAND_UNDER_6_KM = 'REISEAVSTAND_UNDER_6_KM',
+    IKKE_I_TILTAK = 'IKKE_I_TILTAK',
+    IKKE_RETT_TIL_YTELSE = 'IKKE_RETT_TIL_YTELSE',
```

Comment:
Disse to årsakene bør fjernes i backend og her fordi de blir duplikater av ingen målgruppe og ingen aktivitet